src/Seq.Input.RabbitMQ/RabbitMQListener.cs

```diff
@@ -24,11 +27,20 @@ class RabbitMQListener : IDisposable
             var factory = new ConnectionFactory
             {
                 HostName = rabbitMQHost,
+                VirtualHost = rabbitMQVHost,
                 Port = rabbitMQPort,
                 UserName = rabbitMQUser,
                 Password = rabbitMQPassword
             };
-
+            if (IsSsl)
+            {
+                factory.Ssl.AcceptablePolicyErrors = 
+                    SslPolicyErrors.RemoteCertificateNameMismatch |
+                    SslPolicyErrors.RemoteCertificateChainErrors |
+                    SslPolicyErrors.RemoteCertificateNotAvailable;
```

Comment:
Normally these errors can be corrected without giving up the security benefits of SSL by either updating the server with a valid certificate, or adding the server's self-signed certificate to the local trust store on the Seq host machine (slightly different requirements on Windows and Docker).

Because of this we've so far avoided shipping any support for ignoring SSL problems in the Seq client libraries or apps - we view them as potential future security headaches so we're pretty keen to avoid doing so, especially when we can help users tighten up or fix security issues in the process.

Do either of those options (fixing the cert, or adding it to the trust store) work for you?

Reply:
I agree that this is not good. Unfortunately I'm not able to resolve the cert issues for now.

Reply:
Totally understood, thanks for looping back.

Is there a chance you can deploy from a fork, so that we could accept the other changes for now, without the SSL policy configuration? Although we know it's useful in some situations, we're keen to avoid the risk of a user misconfiguring the app and causing a security problem.

Reply:
Yes, I've deployed from my fork. Feel free to remove the SSL policy configuration at your end. When we resolve the issues we'll switch back to your version. Thanks for the tip.